### ISSUE TITLE
Issues driving 32x64 matrix

### ISSUE THREAD

**Issue by rogerguess** [hzeller/rpi-rgb-led-matrix#15]

**Comment by hzeller** [comment]
So without any information it is hard to help you :)

**Comment by rogerguess** [comment]
Ouch... I still have chrome window open where I am drafting my issue, and waiting on a youtube example to upload. I have not clicked 'submit new issue' yet. Sorry, not sure what I did... full description coming.

**Comment by rogerguess** [comment]
i'll close this and start over.
